src/common/snippets/src/pass/mha_tokenization.cpp

```diff
@@ -344,45 +344,6 @@ ov::snippets::pass::TokenizeMHASnippets::TokenizeMHASnippets(const SnippetsToken
          *                  Transpose3
          */
 
-        // First input branch of MatMul0 should be executed before second input branch of MatMul0,
-        // so firstly we insert Transpose1 on the beginning of ordered_ops and then Transpose0
-        // Note: If MatMul0 has transposed_b, we should tokenize only scalars ops from 1st branch
-        //       to move extracted Transpose from MatMul input to body Parameter
-        auto parent = matmul0->get_input_node_shared_ptr(1);
-        // We can support several ops between MatMul0 with transposed_b and Transpose1 with 0213 order (or without this Transpose1)
-        // only if these ops have scalar shapes on other inputs.
-        // There is transformation ExplicitTransposeMatMulInputs that set supported order and transposed_b(false).
-        // We can allow to call this pass only if ops have scalar shapes to avoid shape mismatching
-        const auto is_transposed_b_0 = matmul0->get_transpose_b();
-        bool has_matmul0_has_ops_on_input = false;
-        while (is_supported_intermediate_op(parent)) {
```

Comment:
Do I understand correctly that all intermediate ops will be executed as stand-alone nodes before external repacking? It's interesting to gather statistics on how many models actually have these intermediate ops?
And does this change affects performance of these models?
This information would be useful for future activities like parallel_loops, so we could prioritize particular functionality more efficiently.

Reply:
> Do I understand correctly that all intermediate ops will be executed as stand-alone nodes before external repacking?

Yes, you do. They're executed as Plugin operation in CPU Graph.

> It's interesting to gather statistics on how many models actually have these intermediate ops?

Intermediate ops are usually Eltwise ops with scalar constant. Thanks to "MoveEltwiseUpTrhoughMovement" pass they are moved before `Transpose1`. It means that we usually have MHA Patterns without these intermediate ops

The model list and link to perf validation is attached to the ticket https://jira.devtools.intel.com/browse/CVS-99737

Thanks